scripts/generate-test-certs.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/hashicorp/vault-plugin-auth-pcf/testing/certificates"
+)
+
+const (
+	orgID      = "34a878d0-c2f9-4521-ba73-a9f664e82c7bf"
+	appID      = "2d3e834a-3a25-4591-974c-fa5626d5d0a1"
+	spaceID    = "3d2eba6b-ef19-44d5-91dd-1975b0db5cc9"
+	instanceID = "1bf2e7f6-2d1d-41ec-501c-c70"
+	ipAddr     = "10.255.181.105"
+)
+
+func main() {
+	testCerts, err := certificates.Generate(instanceID, orgID, spaceID, appID, ipAddr)
```

Comment:
This script is added as a tool for developers should they need to generate new test certs for some reason. I needed it for myself anyways and decided to leave it behind for future maintenance.